### PR TITLE
Fix iiif validator errors for media object manifest generation

### PR DESCRIPTION
**Pull request by Dananji**
Related issue: #5798

### CODE REVIEW

File: app/models/iiif_manifest_presenter.rb [hunk 1]
```diff
@@ -211,7 +211,7 @@ def thumbnail_url
       if video_count > 0
         Rails.application.routes.url_helpers.thumbnail_master_file_url(master_file_id)
       elsif audio_count > 0
-        ActionController::Base.helpers.asset_url('audio_icon.png')
+        Rails.application.routes.url_helpers.thumbnail_master_file_url(master_file_id)
```
Comment (cjcolvar) [inline app/models/iiif_manifest_presenter.rb:214]: Why was this change needed?  I think before the goal was to link directly to the asset and avoid having to have another request to rails to get to the asset.

Comment (Dananji) [reply app/models/iiif_manifest_presenter.rb:214]: According to the IIIF 3.0, `id` property must be a URI. And the validator was failing on this.

Comment (Dananji) [reply app/models/iiif_manifest_presenter.rb:214]: Thanks for the suggestion Chris! I tried it (updated the initializer and re-started docker) and still got the relative path. So I tried the following;
```
URI.join(Rails.application.routes.url_helpers.root_url, ActionController::Base.helpers.asset_url('audio_icon.png'))
```
which gave the asset URI instead of the relative path. Do you think this could work?

Comment (cjcolvar) [reply app/models/iiif_manifest_presenter.rb:214]: Interesting.  I wonder if my suggestion didn't work because it runs after ActionController gets loaded.  In a rails console I'm able to make things work if I manually set `asset_host` on `ActionController::Base`:
```
ActionController::Base.asset_host = Rails.application.routes.url_helpers.root_url
```
If you put that in the initializer instead does it work?

Comment (Dananji) [reply app/models/iiif_manifest_presenter.rb]: `Rails.application.routes.url_helpers.root_url` came up as an undefined variable when spinning up the docker container, but combining the variable name and the value from the 2 comments it worked 🎉 
```
ActionController::Base.asset_host = URI("#{Settings.domain.protocol}://#{Settings.domain.host}:#{Settings.domain.port}").to_s
```

**Review by cjcolvar — DISMISSED** [review]
